Library/Homebrew/tab.rb

```diff
@@ -250,6 +250,11 @@ def to_s
     when true  then s << "Poured from bottle"
     when false then s << "Built from source"
     end
+    unless time.nil?
+      s << "Installed" if s.empty?
```

Comment:
I went with @xu-cheng's suggestion because I can't think of a third way a formula could have been installed other than "poured from bottle" or "built from source", even for backward compatibility purposes.